sktime/performance_metrics/forecasting/_classes.py

```diff
@@ -1828,7 +1844,147 @@ class GeometricMeanAbsoluteError(BaseForecastingErrorMetricFunc):
     0.7000014418652152
     """
 
-    func = geometric_mean_absolute_error
+    def _compute_pseudo_values(self, y_true, y_pred):
+        """Compute the jackknife pseudo-values for the 
+        Geometric Mean Absolute Error (GMAE) metric.
+
+        This private method computes the jackknife pseudo-values for the GMAE metric.
+        The pseudo-values are used to estimate the influence of 
+        each observation on the overall metric.
+
+        Parameters
+        ----------
+        y_true : pd.Series or pd.DataFrame
+            Ground truth (correct) target values.
+            y can be in one of the following formats:
+            Series scitype: pd.DataFrame
+            Panel scitype: pd.DataFrame with 2-level row MultiIndex
+            Hierarchical scitype: pd.DataFrame with 3 or more level row MultiIndex
+        y_pred : pd.Series or pd.DataFrame
+            Forecasted values to evaluate
+            must be of same format as y_true, same indices and columns if indexed
+
+        Returns
+        -------
+        pseudo_values : pd.Series or pd.DataFrame
+            Jackknife pseudo-values for the GMAE metric.
+            - If self.multioutput="uniform_average", returns a pd.Series where each 
+              entry is the pseudo-value for a time point, averaged over variables.
+            - If self.multioutput="raw_values", returns a pd.DataFrame where each entry
+              is the pseudo-value for a time point, for each variable.
+        """
+
+        raw_values = (y_true - y_pred).abs()
+
+        n = raw_values.shape[0]
+        gmae_values = []
+        for i in range(n):
```

Comment:
can you vectorize this?

Reply:
while I wrote the code for PR #6461, I saw this as an error and vectorized this since without vectorizing the `gmae_values` it was difficult to multiply it with the custom weights in case of `multioutput` parameter.

Could you please check #6461 once? I think I should close this PR as I have written the correct code taking in reference from this PR